Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM golang:1.18-alpine AS builder
+RUN apk update && apk add --no-cache make git gcc libc-dev linux-headers python3-dev
+WORKDIR /nucleus
+COPY . /nucleus
+RUN rm -rf .git && git init
```

Comment:
Why is this needed?

Reply:
VCS stamping fails if there is no `.git` folder, this was a workaround. It is because the `.git` is a file, not a folder when its a submodule.

Reply:
Setting `GOFLAGS=-buildvcs=false` env should disable VCS.

Reply:
> I think this is part of the "daemon version validation" consensus

I wonder if initializing fresh git context will cause any problem. Maybe there's a better solution instead of removing the actual git state?